lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,40 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * @param {Element} node
+ * @returns {string}
+ */
+/* istanbul ignore next */
+function getNodeTitle(node) {
+  if (node.tagName === 'HTML' || node.tagName === 'BODY') {
+    // too broad, contains all page content
+    return null;
+  }
+  const title = node.innerText || node.getAttribute('alt') || node.getAttribute('aria-label');
+  if (title) {
+    return truncate(title, 80);
+  } else {
+    const nodeToUseForTitle = node.querySelector('[alt], [aria-label]');
+    if (nodeToUseForTitle) {
+      return getNodeTitle(nodeToUseForTitle);
+    }
+  }
```

Comment:
intentionally `undefined` if this all falls through?

Reply:
Yeah, explicitly returning null now.